source/Cosmos.Debug.Hosts/Qemu.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿/*
+using System;
```

Comment:
Can you find a fix which doesnt require commenting out all of Qemu? Or is the file no longer needed?

Reply:
its used in the current master

Reply:
We can just un-comment the whole file when it comes time to merge